crates/red_knot_python_semantic/resources/mdtest/binary/instances.md

```diff
@@ -268,23 +268,28 @@ reveal_type(B() + B())  # revealed: Unknown | int
 
 ## Integration test: numbers from typeshed
 
+We get less precise results from binary operations on float/complex literals due to the special case
+for annotations of `float` or `complex`, which applies also to return annotations for typeshed
+dunder methods. Perhaps we could have a special-case on the special-case, to exclude these typeshed
+return annotations from the widening, and preserve a bit more precision here?
```

Comment:
Should we create and link to an issue?

Reply:
I'm not confident enough that this is worth doing or necessary to justify an issue, or even a TODO; I felt like a mention in the mdtest was the right level of preserving some context for future discussion, in case it comes up. (In other words, I don't feel it's necessary to track this as something we need to remember to come back to, it's more an "if it comes up in the real world" situation.)